Integrations/src/main/java/io/deephaven/integrations/python/PythonMergedListenerAdapter.java

```diff
@@ -116,6 +116,8 @@ protected void propagateErrorDownstream(boolean fromProcess, @NotNull Throwable
                 // and continue with the original exception.
                 log.error().append("Python on_error callback failed: ").append(e2).endl();
             }
+        } else {
+            log.error().append("Python on_error callback is None: ").append(ExceptionUtils.getStackTrace(error)).endl();
```

Comment:
Actually, maybe this whole block is too complicated? We don't allow None/null, right?

Reply:
Yeah, this is more defensive coding than needed but it could server some purposed for someone else who needs to add other types of listeners in the future.